edi_storage_oca/models/edi_exchange_record.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class EDIExchangeRecord(models.Model):
+    _inherit = "edi.exchange.record"
+
+    def _get_dirs(self):
```

Comment:
please:
1. move it to edi.backend
2. rename it to `_storage_get_dirs`
3. pass the direction to it